bootstrap/Old_BootStrapBackup/shoppingmall_seed.py

```
@@ -381,6 +448,12 @@ def seed_core_data(db) -> SeedState:
 
     products: list[Product] = []
     for i, name in enumerate(PRODUCT_NAMES, start=1):
+        image_spec = PRODUCT_IMAGE_SPECS[i]
+        if image_spec.name != name:
+            raise ValueError(
+                f"상품 이미지 매핑 불일치: id={i}, seed={name!r}, image_spec={image_spec.name!r}"
+            )
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)